src/Tribe/Main.php

```diff
@@ -3933,24 +3933,24 @@ public function get_closest_event( $post, $mode = 'next' ) {
 			$start_date = get_post_meta( $post->ID, '_EventStartDate', true );
 			$this->_where_start_date = $start_date;
 
-			$args       = array(
-				'post__not_in'   => array( $post->ID ),
-				'meta_key'       => '_EventStartDate',
-				'orderby'        => array( '_EventStartDate' => $order, 'ID' => $order ),
+			$args = [
+				'post__not_in'   => [ $post->ID ],
+				'orderby'        => [ 'start_date_clause' => $order, 'ID' => $order ],
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 65a9e37fd9874f8aff4ea2c066dfabac</sub>